streaming/src/main/java/io/gearpump/streaming/javaapi/Processor.java

```diff
@@ -36,6 +43,41 @@ public Processor(Class<T> taskClass, int parallelism) {
   }
 
   /**
+   * Create a Source Processor
+   * @param dataSink
+   * @param parallelism
+   * @param description
+   * @param taskConf
+   * @param system
+   * @return
+   */
+  public static Processor<DataSinkTask> sink(DataSink dataSink, int parallelism, String description,  UserConfig taskConf, ActorSystem system) {
```

Comment:
It depends on the scala interface DataSinkProcessor which requires an ActorSystem. If we want to do this, then we need to change DataSinkProcessor first, then maybe changing UserConfig serialization mechanism eventually. It will be a big change. 

I can live with current method, as long as it is compatible with the scala interface(implicit in scala should not be considered as a escape).